src/EigenvalueSolverTypes.f90

```diff
@@ -78,7 +78,10 @@ MODULE EigenvalueSolverTypes
 #endif
 
 !> set enumeration scheme for TPLs
-INTEGER(SIK),PARAMETER,PUBLIC :: SLEPC=0,ANASAZI=1,NATIVE=4
+INTEGER(SIK),PARAMETER,PUBLIC :: SLEPC=0,ANASAZI=1,EV_NATIVE=4
```

Comment:
So we should really give all of these the same prefix.  That's closer to our standard practice.  So `EV_SLEPC`, `EV_ANASAZI`.  May need a full set of temporary parameters to go along with those.

Reply:
Still need to update these.  Just leave the old names there too since I miagine they're used in MPACT.  We can clear out the old names in favor of `EV_*` once MPACT is using thmem.

Reply:
The only one used is NATIVE, actually, which is why it was the only one I left alone. I tested MPACT with these changes and it worked fine. I'll follow your recommendations, but it's not necessary for the eigenvalue TPLs.